source/installation-guide/wazuh-indexer/step-by-step.rst

```diff
@@ -184,11 +190,18 @@ Cluster initialization involves configuring a default ISM policy, loading new ce
 Testing the cluster installation
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-#. Replace ``<WAZUH_INDEXER_IP_ADDRESS>`` and run the following commands to confirm that the installation is successful.
+#. Replace ``<INDEXER_USERNAME>``, ``<INDEXER_PASSWORD>`` and ``<WAZUH_INDEXER_IP_ADDRESS>`` with the Wazuh indexer username and password that you set using the Keystore tool commands:
+
+   .. code-block:: console
+
+      # /var/ossec/bin/wazuh-keystore -f indexer -k username -v <INDEXER_USERNAME>
+      # /var/ossec/bin/wazuh-keystore -f indexer -k password -v <INDEXER_PASSWORD>
+         
+  And run the following commands to confirm that the installation is successful.
```

Comment:
Missing indentation for a good format
```suggestion
    And run the following commands to confirm that the installation is successful.
```

Reply:
Done in [Make requested changes](https://github.com/wazuh/wazuh-documentation/pull/7000/commits/9b2298127240e77d2df7ffd2b3e3f96e5a990cb1) commit